### PR TITLE
Jufa/sle 22848 update gnome

### PR DESCRIPTION
**Pull request by jfaltenbacher**
Update GNOME user guide to 4.1.

* jsc#SLE-22848

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 SP4/openSUSE Leap 15.4 *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [ ] SLE 15 GA
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by jfaltenbacher** [comment]
adding PDF to review chapter 1-16
[book-gnome-user_color_en_chapter_1_16.pdf](https://github.com/SUSE/doc-sle/files/8463575/book-gnome-user_color_en_chapter_1_16.pdf)

**Comment by jfaltenbacher** [comment]
Hi @dariavladykina please check my changes. You can also check in the PDF attached for better reading. Screenshots will follow. Also the remaining chapters will follow in the next days.

File: xml/apps_evolution.xml [hunk 1]
```diff
@@ -17,33 +23,34 @@
...
   can easily reuse appointment or contact information sent to you by e-mail, or
   send e-mails to a contact or appointment. If you receive lots of e-mail, you
   can use advanced features like search folders, which let you save searches as
   though they were ordinary e-mail folders.
  </para>
  <para>
-  This chapter introduces you to Evolution and helps you get started. For more
-  details, refer to the Evolution application help.
+  This chapter introduces you to <productname>Evolution</productname> and helps
+  you get started. For more details, refer to the application help.
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:46]:
```suggestion
  you get started. For more information, refer to the application help.
```

File: xml/apps_evolution.xml [hunk 2]
```diff
@@ -58,16 +65,19 @@
...
-   The first time you start Evolution, it opens an assistant to help you set up
-   e-mail accounts and import data from other applications.
+   The first time you start <productname>Evolution</productname>, it opens an
+   assistant to help you set up e-mail accounts and import data from other
+   applications.
   </para>
 
   <para>
-   The <guimenu>Evolution Account Assistant</guimenu> helps you provide all the
+   The <guimenu>Evolution Accounts</guimenu> Assitant helps you provide all the
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:74]:
```suggestion
   The <guimenu>Evolution Accounts</guimenu> Assistant helps you provide all the
```

File: xml/apps_evolution.xml [hunk 3]
```diff
@@ -17,33 +23,34 @@
...
+  <productname>Evolution</productname> helps you to work in a group by
+  handling e-mail, contact information, and one or more calendars. It can do
+  that on one or several computers, connected directly or over a network, for
+  one person or for large groups.
  </para>
  <para>
-  Evolution helps you accomplish common daily tasks quickly. For example, you
+  It also helps you accomplish common daily tasks quickly. For example, you
   can easily reuse appointment or contact information sent to you by e-mail, or
   send e-mails to a contact or appointment. If you receive lots of e-mail, you
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:40]:
```suggestion
  send e-mails to a contact or appointment. If you receive lots of e-mails, you
```

File: xml/apps_evolution.xml [hunk 4]
```diff
@@ -128,10 +135,10 @@
    <title>Receiving mail</title>
    <para>
     The <guimenu>Receiving E-mail</guimenu> page lets you determine the server
-    that you want to use to receive e-mail.
+    that you want to use to receive e-mails.
    </para>
    <para>
-    You need to specify the type of server you want to receive mail from. If
+    You need to specify the type of server you want to receive mails from. If
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:141]:
```suggestion
    You need to specify the type of server you want to receive e-mails from. If
```

File: xml/apps_evolution.xml [hunk 5]
```diff
@@ -772,12 +780,11 @@
   </sect2>
 
   <sect2 xml:id="sec-gnome-evolution-first-start-setup-assistant-sending">
-   <title>Sending mail</title>
+   <title>Sending Email</title>
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:783]:
```suggestion
   <title>Sending mail</title>
```

Comment (dariavladykina) [reply xml/apps_evolution.xml:783]: For consistency with 'Receiving mail'.

File: xml/apps_evolution.xml [hunk 6]
```diff
@@ -336,9 +344,9 @@
     preferences for its behavior.
    </para>
    <sect3 xml:id="sec-gnome-evolution-first-start-setup-assistant-receiving-opts-imap">
-    <title>IMAP+ receiving options</title>
+    <title>IMAP receiving options</title>
     <para>
-     If you selected IMAP+ as the receiving server type, you will now see a
+     If you selected IMAP+as the receiving server type, you will now see a
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:349]:
```suggestion
     If you selected IMAP as the receiving server type, you will now see a
```

File: xml/apps_evolution.xml [hunk 7]
```diff
@@ -772,12 +780,11 @@
...
 
   <sect2 xml:id="sec-gnome-evolution-first-start-setup-assistant-sending">
-   <title>Sending mail</title>
+   <title>Sending Email</title>
    <para>
     Now that you have entered information about how you plan to receive mail,
-    Evolution needs to know about how you want to send it. Usually, a separate
-    server configuration is necessary for this.
-<!-- Exchange: - sknorr, 2015-09-21 -->
+    <productname>Evolution</productname> needs to know about how you want to
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:786]:
```suggestion
    <productname>Evolution</productname> needs to know how you want to
```

File: xml/apps_evolution.xml [hunk 8]
```diff
@@ -906,7 +914,8 @@
     <term>Menu bar</term>
     <listitem>
      <para>
-      The menu bar gives you access to nearly all of the features of Evolution.
+      The menu bar gives you access to nearly all of the features of
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:917]:
```suggestion
      The menu bar gives you access to nearly all the features of
```

File: xml/apps_evolution.xml [hunk 9]
```diff
@@ -1042,21 +1051,20 @@
...
-    search folders.
+    component has at least one, called <guimenu>On This Computer</guimenu>, for
+    local information. For example, the folder list for the e-mail component
+    shows all your e-mail accounts, local folders, and search folders.
    </para>
    <para>
     If you receive large amounts of e-mail, you need additional ways to
-    organize it. In Evolution, you can create own e-mail folders, address
-    books, calendars, task lists, or memo lists.
+    organize it. You can create own e-mail folders, address books, calendars,
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:1066]:
```suggestion
    organize it. You can create your own e-mail folders, address books,
```

File: xml/apps_evolution.xml [hunk 10]
```diff
@@ -1288,8 +1292,8 @@
    <title>Managing contacts</title>
    <para>
     To use the contacts component, click <guimenu>Contacts</guimenu> in the
-    shortcut bar. The Evolution contacts component can handle all of the
-    functions of an address book or phone book.
+    shortcut bar. The <productname>Evolution</productname> contacts component
+    can handle all of the functions of an address book or phone book.
```
Comment (dariavladykina) [inline xml/apps_evolution.xml:1296]:
```suggestion
    can handle all the functions of an address book or phone book.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, here are some comments for Ekiga and Evolution.

File: xml/apps_firefox.xml [hunk 11]
```diff
@@ -1016,11 +1041,11 @@
...
      configuration options can be accessed via the
      <guimenu>Preferences</guimenu> button. In the
-     <guimenu>Appearance</guimenu> tab you may <guimenu>Uninstall</guimenu> a
+     <guimenu>Themes</guimenu> tab you may <guimenu>Uninstall</guimenu> a
      theme, or activate a different theme by clicking
      <guimenu>Enable</guimenu>. Pending extension and theme installations are
      also listed. Select <guimenu>Cancel</guimenu> to stop the installation.
-     Although you cannot install <guimenu>Plug-Ins</guimenu> as a user, you may
+     Although you cannot install <guimenu>Plugins</guimenu> as a user, you may
      disable or enable them with the Add-ons manager.
```
Comment (dariavladykina) [inline xml/apps_firefox.xml:1049]:
```suggestion
     disable or enable them with the Add-ons Manager.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, some corrections/suggestions for Firefox chapter.

File: xml/apps_evolution.xml (hunk 5 above)
Comment (jfaltenbacher) [reply xml/apps_evolution.xml:783]: changed to E-mail for both occurences

File: xml/apps_firefox.xml [hunk 12]
```diff
@@ -281,22 +290,23 @@
...
      search Wikipedia for <replaceable>SEARCHTERM</replaceable>.
     </para>
     <para>
      To assign a shortcut for a search engine from the search bar, click the
-     icon to the left of the search bar and select <guimenu>Change Search
-     Settings</guimenu>. Select a search engine, double-click its
-     <guimenu>Keyword</guimenu> column, enter a keyword and press
-     <keycap
-      function="enter"></keycap>.
+     the hamburger icon to the right of the search bar and select <menuchoice>
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/apps_firefox.xml:302]: changed as suggested

File: xml/apps_firefox.xml [hunk 13]
```diff
@@ -778,82 +799,88 @@
...
    <para>
     To review or manage your passwords, open the password manager by clicking
-    <menuchoice> <guimenu>Edit</guimenu> <guimenu>Preferences</guimenu>
-    <guimenu>Security</guimenu> <guimenu>Saved Passwords</guimenu>
-    </menuchoice> in the menu bar. The password manager opens with a list of
-    sites and their corresponding user names. By default, the passwords are not
-    displayed. You can click <guimenu>Show Passwords</guimenu> to display them.
+    <menuchoice> <guimenu>Edit</guimenu> <guimenu>Settings</guimenu>
+    <guimenu>Privacy &amp; Security</guimenu> <guimenu>Logins and Passwords
+    </guimenu> <guimenu>Saved logins</guimenu></menuchoice> in the menu bar.
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/apps_firefox.xml:814]: changed as suggested

File: xml/gnome_crypto.xml [hunk 14]
```diff
@@ -1132,26 +1083,51 @@
...
+     Press the <guimenu>Key Servers</guimenu> button to see the list of key
+     servers.
     </para>
    </step>
    <step>
     <para>
-     Select <guimenu>Automatically synchronize modified keys with key
-     servers</guimenu>.
+     To publish your key, select a server from the menu. Close the window and
+     go back to the previuos dialog.
```
Comment (dariavladykina) [inline xml/gnome_crypto.xml:1125]:
```suggestion
     go back to the previous dialog.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Julia, please take a look at some correction to several more files. Thank you!

File: xml/gnome_crypto.xml [hunk 15]
```diff
@@ -860,54 +794,63 @@
...
-      <imageobject role="html">
-       <imagedata fileref="seahorse_signkey.png" width="85%" format="PNG"/>
-      </imageobject>
-     </mediaobject>
-    </informalfigure>
    </step>
    <step>
     <para>
-     Click <guimenu>Sign</guimenu>.
+     Decide if you want to revoke your signature ar a later date and if you want
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/gnome_crypto.xml:835]: because you can really select a date here

File: xml/gnome_print.xml [hunk 16]
```diff
@@ -166,47 +175,37 @@
...
+      <literal>Printers</literal>.
      </para>
      <para>
-      The printer dialog should open. If it does not, click <menuchoice>
-      <guimenu>Computer</guimenu> <guimenu>Control Center</guimenu>
-      <guimenu>Hardware</guimenu> <guimenu>Printing</guimenu>
-      <guimenu>New</guimenu> <guimenu>Printer</guimenu> </menuchoice> to
-      open it.
+      Click <guimenu>Printers</guimenu>. You may have to press
+      <guimenu>Unlock</guimenu> in the top right corner and type in your
```
Comment (dariavladykina) [inline xml/gnome_print.xml:196]:
```suggestion
      <guimenu>Unlock</guimenu> in the top-right corner and type in your
```

File: xml/gnome_print.xml [hunk 17]
```diff
@@ -216,79 +215,79 @@
...
     <para>
-     Double-click the printer you want to modify.
+    Click <guimenu>Printers</guimenu> to open the panel.
     </para>
    </step>
    <step>
     <para>
-     Modify the properties and click <guimenu>Close</guimenu>.
+     Depending on your system, you may have to press <guimenu>Unlock</guimenu>
+     in the top right corner type in your
```
Comment (dariavladykina) [inline xml/gnome_print.xml:237]:
```suggestion
     in the top-right corner type in your
```

File: xml/gnome_start.xml [hunk 18]
```diff
@@ -492,38 +465,38 @@
...
   <title>Pausing or finishing your session</title>
 
   <para>
-   When you have finished using the computer, there are multiple ways to finish
-   the session. Which one is right in a given situation depends on how long you
-   will be away and whether you are worried about energy consumption, among
-   other things.
-  </para>
+   To pause or close your session, open the <guimenu>System</guimenu> menu
+   in the upper right corner of the top bar and choose one of the options
```
Comment (dariavladykina) [inline xml/gnome_start.xml:481]:
```suggestion
   in the top-right corner of the top bar and choose one of the options
```

**Review by dariavladykina — COMMENTED** [review]
Hi Julia, another portion of comments. Thank you!

File: xml/apps_libreoffice.xml [hunk 19]
```diff
@@ -658,38 +650,37 @@
...
+     clicking <guimenu>Import</guimenu> and then locate your certificate.
     </para>
    </step>
   </procedure>
   <para>
    To sign a document, first open it in &libo;. Then select
    <menuchoice><guimenu>File</guimenu><guimenu>Digital Signatures</guimenu>
-   <guimenu>Sign Document</guimenu></menuchoice>. Select the certificate you
-   want to use for signing, then click <guimenu>OK</guimenu>.
+   <guimenu>Digital Signatures</guimenu></menuchoice>. Select the certificate
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/apps_libreoffice.xml:681]: yes

File: xml/apps_totem.xml [hunk 20]
```diff
@@ -349,40 +381,39 @@
...
      </listitem>
     </varlistentry>
     <varlistentry>
-     <term>Text subtitles</term>
+     <term>Plugins</term>
      <listitem>
       <para>
-       Lets you specify whether to load the subtitles automatically, and change
-       the font and encoding used to display the subtitles.
+       Plugins offer extra featured and extend functionality. You can view the
```
Comment (dariavladykina) [inline xml/apps_totem.xml:404]:
```suggestion
       Plugins offer extra features and extend functionality. You can view the
```

File: xml/apps_totem.xml [hunk 21]
```diff
@@ -349,40 +381,39 @@
...
 
   <sect2 xml:id="sec-gnome-totem-prefs-display">
-   <title>Display preferences</title>
+   <title><guimenu>Display</guimenu> preferences</title>
    <para>
-    The Display Preferences let you choose to automatically resize the window
-    when a new video is loaded, change the color balance, and configure visual
-    effects when an audio file is played.
+    The Display Preferences let you choose if you want to enable deinterlacing
+    and allows you to adjust Brightness, Contrast, Saturation or Hue.
```
Comment (dariavladykina) [inline xml/apps_totem.xml:416]:
```suggestion
    and allow you to adjust Brightness, Contrast, Saturation or Hue.
```

File: xml/apps_totem.xml [hunk 22]
```diff
@@ -400,46 +431,29 @@
...
-    </varlistentry>
-    <varlistentry>
-     <term>Visual effects</term>
+     <term><guimenu>Videos</guimenu></term>
      <listitem>
       <para>
-       You can choose to show visual effects when an audio file is playing,
-       select the type of visualization you want to show, and the visualization
-       size.
+      Allows you to specify, whether you want to <guimenu>Disable deinterlacing
```
Comment (dariavladykina) [inline xml/apps_totem.xml:437]:
```suggestion
      Allows you to specify whether you want to <guimenu>Disable deinterlacing
```

File: xml/apps_totem.xml [hunk 23]
```diff
@@ -400,46 +431,29 @@
...
     </varlistentry>
     <varlistentry>
-     <term>Color balance</term>
+     <term><guimenu>Color balance</guimenu></term>
      <listitem>
       <para>
-       Specify the level of color brightness, contrast, saturation, and hue.
+       This option allows you to adjust <guimenu>Brightness</guimenu>,
+       <guimenu>Contrast</guimenu>, <guimenu>Saturation</guimenu> or
+       <guimenu>Hue</guimenu> of viewed videos.
```
Comment (dariavladykina) [inline xml/apps_totem.xml:448]:
```suggestion
       <guimenu>Hue</guimenu> of your videos.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Julia, another portion of checks. Thanks!

File: xml/apps_firefox.xml [hunk 24]
```diff
@@ -578,35 +598,33 @@
...
    <para>
-    The <literal>Bookmarks Toolbar</literal> is displayed beneath the location
+    The <guimenu>Bookmarks Toolbar</guimenu> is displayed beneath the address
     bar and lets you quickly access bookmarks. You can also add, organize, and
-    edit bookmarks directly. By default, the <literal>Bookmarks
-    Toolbar</literal> is populated with a predefined set of bookmarks organized
-    into several folders (see <xref linkend="fig-firefox-main"/>).
+    edit bookmarks directly. To activate the <guimenu>Bookmarks
+    Toolbar</guimenu>, select <menuchoice><guimenu>View </guimenu>
+    <guimenu>Toolbars</guimenu> <guimenu>Bookmarks Toolbar </guimenu>
```
[the remark this answers is not in the document]
Comment (jfaltenbacher) [reply xml/apps_firefox.xml:609]: here the spaces are needed because of the line break.

File: xml/gnome_start.xml [hunk 25]
```diff
@@ -234,112 +243,57 @@
...
  </sect1>
  <sect1 xml:id="sec-gnome-desktop">
   <title>Desktop basics</title>
 
   <para>
-   The &gnome; desktop appears after you first log in. It displays a panel at
-   the bottom showing the following elements (from left to right):
+   The &gnome; desktop appears after you first log in. The top bar displays the
+   <guimenu>Activities</guimenu> Overview, the calendar, and the
+   <guimenu>System</guimenu> menu. The following list, gives more
```
Comment (dariavladykina) [inline xml/gnome_start.xml:262]:
```suggestion
   <guimenu>System</guimenu> menu. The following list gives more
```

**Review by dariavladykina — COMMENTED** [review]
Some more corrections.

File: xml/gnome_start.xml [hunk 26]
```diff
@@ -645,41 +617,35 @@
      </para>
      <variablelist>
       <varlistentry>
-       <term>Power off</term>
+       <term><guimenu>Suspend</guimenu></term>
        <listitem>
         <para>
-         Logs you out of the current session, then turns off the computer.
+         Pause your session and put the computer in a state where it consumes a
```
Comment (dariavladykina) [inline xml/gnome_start.xml:623]:
```suggestion
         Pause your session and put the computer into a state where it consumes a
```

File: xml/gnome_use.xml [hunk 27]
```diff
@@ -4,24 +4,34 @@
...
+  In this chapter you will learn how to perform regular tasks with &gnome;
+  desktop.
  </para>
  <sect1 xml:id="sec-gnomeuser-use-nautilus">
   <title>Managing files and directories</title>
 
   <para>
-   To start &nautilus; open the Activities Overview by pressing <keycap
-   function="meta"/> and search for <literal>files</literal>.
+   To start &nautilus; open the <guimenu>Activities</guimenu> Overview  by
```
Comment (dariavladykina) [inline xml/gnome_use.xml:32]:
```suggestion
   To start &nautilus;, open the <guimenu>Activities</guimenu> Overview  by
```

File: xml/gnome_use.xml [hunk 28]
```diff
@@ -340,11 +335,11 @@
...
-    In all these cases, you can create a compressed file, such as a ZIP file,
-    which can contain multiple original files. How much smaller the compressed
-    version is than the original depends on the file type. Many video, image
-    and office document formats are already compressed and will only become
-    marginally smaller.
+    In all these cases, you can create a compressed file, such as a
+    <filename>ZIP</filename> file, which can contain multiple original files.
+    How much smaller the compressed version is than the original depends on the
+    file type. Many video, image and office document formats are already
+    compressed and will only become marginally smaller.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:342]:
```suggestion
    How much smaller the compressed version is compared to the original, depends
    on the file type. Many video, image and office document formats are already
    compressed and will only become marginally smaller.
```

File: xml/gnome_use.xml [hunk 29]
```diff
@@ -4,24 +4,34 @@
...
+<chapter xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ version="5.0"
+ xml:id="cha-gnomeuser-use">
  <title>Working with your desktop</title>
  <info>
+  <abstract>
+   <para>
+    Learn how to perform regular tasks with &gnome; desktop.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:16]:
```suggestion
    Learn how to perform regular tasks with the &gnome; desktop.
```

File: xml/gnome_use.xml [hunk 30]
```diff
@@ -746,8 +725,8 @@
   </para>
 
   <para>
-   To start &libo; open the Activities Overview by pressing <keycap
-   function="meta"/> and type <literal>libre</literal>.
+   To start &libo; open the <guimenu>Activities</guimenu> Overview by pressing
```
Comment (dariavladykina) [inline xml/gnome_use.xml:728]:
```suggestion
   To start &libo;, open the <guimenu>Activities</guimenu> Overview by pressing
```

File: xml/gnome_use.xml [hunk 31]
```diff
@@ -758,17 +737,15 @@
   <title>Controlling your desktop’s power management</title>
 
   <para>
-   To see the state of the computer battery on your laptop, check the battery
-   icon in the right part of the panel. On certain events, such as a critically
-   low battery state, &gnome; will display notifications informing you about
-   the event.
+   You can open the power settings by opening the <guimenu>Activities</guimenu>
+   overview by pressing <keycap function="meta"/> and typing
+   <literal>power</literal>. Now choose <menuchoice> <guimenu>Settings</guimenu>
```
Comment (dariavladykina) [inline xml/gnome_use.xml:742]:
```suggestion
   Overview by pressing <keycap function="meta"/> and typing
   <literal>power</literal>. Now select <menuchoice> <guimenu>Settings</guimenu>
```

File: xml/gnome_use.xml [hunk 32]
```diff
@@ -780,28 +757,58 @@
...
-   common formats such as <filename>zip</filename>,
-   <filename>tar.gz</filename>, <filename>tar.bz2</filename>,
-   <filename>lzh</filename>, and <filename>rar</filename>. You can use Archive
-   Manager to create, open and extract a compressed non-archive file.
+   and subdirectories, usually in compressed form.
+   You can use <productname>Archive Manager</productname> to create, open and
+   extract a compressed non-archive file.
   </para>
+  <para>
+   <productname>Archive Manager</productname> supports common formats such as:
```
Comment (dariavladykina) [inline xml/gnome_use.xml:768]:
```suggestion
   <productname>Archive Manager</productname> supports common formats, such as:
```

File: xml/gnome_use.xml [hunk 33]
```diff
@@ -780,28 +757,58 @@
...
+    <para>
+      <filename>rar</filename>
+    </para>
+   </listitem>
+    </itemizedlist>
 
   <para>
-   To start Archive Manager open the Activities Overview by pressing <keycap
-   function="meta"/> and type <literal>arch</literal>.
+   To start <guimenu>Archive Manager</guimenu> open the
```
Comment (dariavladykina) [inline xml/gnome_use.xml:799]:
```suggestion
   To start <guimenu>Archive Manager</guimenu>, open the
```

File: xml/gnome_use.xml [hunk 34]
```diff
@@ -780,28 +757,58 @@
...
+   <guimenu>Activities</guimenu> Overview by pressing
+   <keycap function="meta"/> and type <literal>arch</literal>.
   </para>
 
   <para>
    If you already have a compressed file, double-click the file name in
-   &nautilus; to view the contents of the archive in Archive Manager.
+   &nautilus; to view the contents of the archive in
+   <productname>Archive Manager</productname> .
```
Comment (dariavladykina) [inline xml/gnome_use.xml:807]:
```suggestion
   <productname>Archive Manager</productname>.
```

File: xml/gnome_use.xml [hunk 35]
```diff
@@ -817,20 +824,16 @@
    <procedure>
     <step>
      <para>
-      In Archive Manager, click <guimenu>Open</guimenu>.
+      Open <productname>Archive Manager</productname>, click the menu button
+      in the top-right corner of the window and select <guimenu>Open</guimenu>.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:828]:
```suggestion
      in the upper right corner of the window, and select <guimenu>Open</guimenu>.
```

File: xml/gnome_use.xml [hunk 36]
```diff
@@ -1032,8 +1019,8 @@
    <procedure>
     <step>
      <para>
-      In Archive Manager, click the main menu icon in the top left part of the
-      window and select <guimenu>New Archive</guimenu>.
+      In <productname>Archive Manager</productname>, click the menu button in the
+      top right part of the window and select <guimenu>New Archive</guimenu>.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:1023]:
```suggestion
      upper right part of the window and select <guimenu>New Archive</guimenu>.
```

File: xml/gnome_use.xml [hunk 37]
```diff
@@ -1051,11 +1038,11 @@
...
-      Archive Manager creates an empty archive, but does not yet write the
-      archive to disk. Archive Manager writes a new archive to disk only when
-      the archive contains at least one file. If you create a new archive and
-      quit Archive Manager before you add any files to the archive, the archive
-      will be deleted.
+      <productname>Archive Manager </productname> creates an empty archive, but
+      does not yet write the archive to disk. A new archive is only written to
+      the disk written, when the archive contains at least one file. If you
+      create a new archive and quit <productname>Archive Manager</productname>
+      before you add any files to the archive, the archive will be deleted.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:1045]:
```suggestion
      does not yet write the archive to disk. A new archive is only written to
      the disk when the archive contains at least one file. If you create
      a new archive and quit <productname>Archive Manager</productname>
      before adding any files, the archive will be deleted.
```

File: xml/gnome_use.xml [hunk 38]
```diff
@@ -1093,32 +1081,63 @@
...
-
   <para>
-   The screenshots are automatically saved to your
-   <filename>~/Pictures</filename> directory.
+   Use the following global keyboard shortcuts to quickly take a screenshot:
   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      <keycap>Print</keycap> takes a screenshot of the entire desktop
```
Comment (dariavladykina) [inline xml/gnome_use.xml:1094]:
```suggestion
      <keycap>Print</keycap> takes a screenshot of the entire desktop.
```

File: xml/gnome_use.xml [hunk 39]
```diff
@@ -1145,22 +1164,25 @@
...
+   To open <productname>Document Viewer</productname>, double-click a PDF file
+   in a file manager window. <productname>Document Viewer</productname> will
+   also open when you download a PDF file from a Web site.
+  
   </para>
-
+<para>
+  To open <productname>Document Viewer</productname> without a file, open the
+ <guimenu>Activities</guimenu> Overview by pressing
+ <keycap function="meta"/> and type <literal>pdf</literal>.
```
Comment (dariavladykina) [inline xml/gnome_use.xml:1175]:
```suggestion
 <keycap function="meta"/> and typing <literal>pdf</literal>.
```

**Review by dariavladykina — COMMENTED** [review]
Hi Julia, seems that I'm done with the review. I suggest we do a quick check once more after the book is finalized. Thanks!

**Comment by jfaltenbacher** [comment]
thanks @dariavladykina . I have committed the changes. Can you please approve this PR so I can merge. 
Not sure, if we manage to do another review before the proofing drop - let's see

**Pull request merged**